cdist_manifestation/dependencies.py

```diff
@@ -0,0 +1,15 @@
+from contextlib import contextmanager
+
+_dependencies = []
+
+
+@contextmanager
+def require(*args):
```

Comment:
Shouldn't we support also an equivalent of setting `CDIST_ORDER_DEPENDENCY` variable?

Reply:
We should. `order_dependency()` added.